projects/istio/Dockerfile

```diff
@@ -0,0 +1,20 @@
+# Copyright 2020 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+FROM gcr.io/oss-fuzz-base/base-builder
+RUN go get github.com/istio/istio/pilot/pkg/config/kube/crd/...
```

Comment:
Should this be `go get github.com/istio/istio/...` so it doesn't need to be changed anytime we add a new dependency for the fuzzer?

Reply:
You got it. Thanks!